src/categories/land-management.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Land Management"
+image: ../images/carbon-reduction.jpg
```

Comment:
Could use something from [tree surgery](https://stock.adobe.com/search?filters%5Bcontent_type%3Aphoto%5D=1&filters%5Bcontent_type%3Aillustration%5D=1&filters%5Bcontent_type%3Azip_vector%5D=1&filters%5Bcontent_type%3Avideo%5D=1&filters%5Bcontent_type%3Atemplate%5D=1&filters%5Bcontent_type%3A3d%5D=1&filters%5Bcontent_type%3Aimage%5D=1&k=tree+surgery&order=relevance&safe_search=1&search_page=1&search_type=usertyped&acp=&aco=tree+surgery&get_facets=0)

Or this fella 

![image](https://user-images.githubusercontent.com/67381/69345360-eb2dc980-0c68-11ea-874a-30c65b6e4b46.png)